tests/nnstreamer_filter_extensions_common/meson.build

```diff
@@ -39,12 +39,14 @@ if have_python3
   extensions += [['python3', 'python3', nnstreamer_filter_python3_deps, 'passthrough.py']]
 endif
 
+ext_test_build_dir = meson.current_build_dir()
```

Comment:
As `.cc` files are installed in `build` dir, lets add an exception in `packaging/run_unittest_binaries.sh` to not ignore `.cc` or `.c` files. Otherwise, the below silent errors occur:
```
...
[  128s] ./tests/nnstreamer_filter_extensions_common/unittest_tizen_python2.cc
[  128s] /home/abuild/rpmbuild/BUILD/nnstreamer-1.2.0/packaging/run_unittests_binaries.sh: line 31: ./tests/nnstreamer_filter_extensions_common/unittest_tizen_python2.cc: Permission denied
...
[  128s] ./tests/nnstreamer_filter_extensions_common/unittest_tizen_custom.cc
[  128s] /home/abuild/rpmbuild/BUILD/nnstreamer-1.2.0/packaging/run_unittests_binaries.sh: line 31: ./tests/nnstreamer_filter_extensions_common/unittest_tizen_custom.cc: Permission denied
...
```